pootle/templates/editor/units/xhr_checks.html

```diff
@@ -15,3 +16,16 @@
   {% endfor %}
 </ul>
 {% endif %}
+{% endwith %}
+{% with unit.get_qualitychecks_warning as checks_warning %}
+{% if checks_warning %}
+<span class="sidetitle" lang="{{ LANGUAGE_CODE }}" title="{% trans 'Possible issues with the translation' %}">{% trans "Watch out for:" %}</span>
+<ul class="checks checks-warning">
+  {% for check in checks_warning %}
+  <li class="check{% if check.false_positive %} false-positive{% endif %}">
```

Comment:
Is actually necessary to check for `false_positive` for warning checks that can't be muted?